[Automation] java run fails for org.scalacheck:scalacheck_2.13:1.19.0

## java run on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25469529742/job/74730397222
Last 300 lines of the log:
```console

PlatformTest > frameworkInstantiatesPropertiesClass() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkInstantiatesPropertiesClass(PlatformTest.scala:42)

PlatformTest > frameworkLoadsPropertiesModule() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkLoadsPropertiesModule(PlatformTest.scala:32)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

2 tests completed, 2 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 20s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 7 executed, 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.scalacheck:scalacheck_2.13:1.18.1 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 29s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 1.19.0 at 2026-05-07T00:52:55Z with exit code 1 after 30s
❌ ERROR: 'native-image run' for 1.19.0 failed with exit code 1.
SOFT-FAIL[native-image run][1.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean test -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"]
========================================================================================
 org.scalacheck:scalacheck_2.13:1.18.1:1.19.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 1.19.0 at 2026-05-07T00:52:55Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean compileTestJava -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 871ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 868ms
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 1.19.0 at 2026-05-07T00:52:57Z with exit code 0 after 2s
========================================================================================
 org.scalacheck:scalacheck_2.13:1.18.1:1.19.0 stage java run
========================================================================================
Starting stage 'java run' for 1.19.0 at 2026-05-07T00:52:57Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 828ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:processResources NO-SOURCE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :compileScala NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:jar UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :compileTestScala
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

PlatformTest > frameworkInstantiatesPropertiesClass() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkInstantiatesPropertiesClass(PlatformTest.scala:42)

PlatformTest > frameworkLoadsPropertiesModule() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkLoadsPropertiesModule(PlatformTest.scala:32)


2 tests completed, 2 failed

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
BUILD FAILED in 6s

14 actionable tasks: 4 executed, 10 up-to-date

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 8s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 1.19.0 at 2026-05-07T00:53:06Z with exit code 1 after 9s
❌ ERROR: 'java run' for 1.19.0 failed with exit code 1.
FAILED[java run][1.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"]
```

## java run on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25469529742/job/74730397206
Last 300 lines of the log:
```console

PlatformTest > frameworkInstantiatesPropertiesClass() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkInstantiatesPropertiesClass(PlatformTest.scala:42)

PlatformTest > frameworkLoadsPropertiesModule() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkLoadsPropertiesModule(PlatformTest.scala:32)

2 tests completed, 2 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.


For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 7 executed, 8 up-to-date
BUILD FAILED in 24s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.scalacheck:scalacheck_2.13:1.18.1 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 33s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 1.19.0 at 2026-05-07T00:52:55Z with exit code 1 after 33s
❌ ERROR: 'native-image run' for 1.19.0 failed with exit code 1.
SOFT-FAIL[native-image run][1.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean test -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"]
========================================================================================
 org.scalacheck:scalacheck_2.13:1.18.1:1.19.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 1.19.0 at 2026-05-07T00:52:55Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean compileTestJava -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 848ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 903ms
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 1.19.0 at 2026-05-07T00:52:58Z with exit code 0 after 3s
========================================================================================
 org.scalacheck:scalacheck_2.13:1.18.1:1.19.0 stage java run
========================================================================================
Starting stage 'java run' for 1.19.0 at 2026-05-07T00:52:58Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 933ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileScala NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :compileTestScala
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

PlatformTest > frameworkInstantiatesPropertiesClass() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkInstantiatesPropertiesClass(PlatformTest.scala:42)

PlatformTest > frameworkLoadsPropertiesModule() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkLoadsPropertiesModule(PlatformTest.scala:32)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

2 tests completed, 2 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

BUILD FAILED in 6s

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 8s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 1.19.0 at 2026-05-07T00:53:07Z with exit code 1 after 9s
❌ ERROR: 'java run' for 1.19.0 failed with exit code 1.
FAILED[java run][1.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"]
```

## java run on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25469529742/job/74730397179
Last 300 lines of the log:
```console

PlatformTest > frameworkInstantiatesPropertiesClass() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkInstantiatesPropertiesClass(PlatformTest.scala:42)

PlatformTest > frameworkLoadsPropertiesModule() FAILED

2 tests completed, 2 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 23s
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkLoadsPropertiesModule(PlatformTest.scala:32)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 7 executed, 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.scalacheck:scalacheck_2.13:1.18.1 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 31s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 1.19.0 at 2026-05-07T00:52:49Z with exit code 1 after 32s
❌ ERROR: 'native-image run' for 1.19.0 failed with exit code 1.
SOFT-FAIL[native-image run][1.19.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.19.0" ./gradlew clean test -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"]
========================================================================================
 org.scalacheck:scalacheck_2.13:1.18.1:1.19.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 1.19.0 at 2026-05-07T00:52:49Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.19.0" ./gradlew clean compileTestJava -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 908ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :compileScala NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 965ms
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 1.19.0 at 2026-05-07T00:52:52Z with exit code 0 after 3s
========================================================================================
 org.scalacheck:scalacheck_2.13:1.18.1:1.19.0 stage java run
========================================================================================
Starting stage 'java run' for 1.19.0 at 2026-05-07T00:52:52Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 829ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :compileScala NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestScala
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

PlatformTest > frameworkInstantiatesPropertiesClass() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)

2 tests completed, 2 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 6s
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkInstantiatesPropertiesClass(PlatformTest.scala:42)

PlatformTest > frameworkLoadsPropertiesModule() FAILED
    org.opentest4j.AssertionFailedError: expected: <1> but was: <0>
        at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
        at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
        at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)
        at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:528)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.runDiscoveredProperties(PlatformTest.scala:64)
        at app//org_scalacheck.scalacheck_2_13.PlatformTest.frameworkLoadsPropertiesModule(PlatformTest.scala:32)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.scalacheck/scalacheck_2.13/1.18.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 8s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 1.19.0 at 2026-05-07T00:53:01Z with exit code 1 after 9s
❌ ERROR: 'java run' for 1.19.0 failed with exit code 1.
FAILED[java run][1.19.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.19.0" ./gradlew clean javaTest -Pcoordinates="org.scalacheck:scalacheck_2.13:1.18.1"]
```

--- a/metadata/org.scalacheck/scalacheck_2.13/1.19.0/reachability-metadata.json
+++ b/metadata/org.scalacheck/scalacheck_2.13/1.19.0/reachability-metadata.json
@@ -1,0 +1,52 @@
+{
+  "reflection": [
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.rng.Seed$"
+      },
+      "type": "java.lang.Object[][]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$2"
+      },
+      "type": "java.lang.String[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$3"
+      },
+      "type": "java.lang.String[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.Platform$"
+      },
+      "type": "org_scalacheck.scalacheck_2_13.PlatformClassProperties"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.Platform$"
+      },
+      "type": "org_scalacheck.scalacheck_2_13.PlatformModuleProperties$"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.ScalaCheckRunner"
+      },
+      "type": "sbt.testing.Task[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$2"
+      },
+      "type": "sbt.testing.Task[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$3"
+      },
+      "type": "sbt.testing.Task[]"
+    }
+  ]
+}

--- a/metadata/org.scalacheck/scalacheck_2.13/1.19.0/reachability-metadata.json
+++ b/metadata/org.scalacheck/scalacheck_2.13/1.19.0/reachability-metadata.json
@@ -1,52 +1,40 @@
 {
-  "reflection": [
+  "reflection" : [
     {
-      "condition": {
-        "typeReached": "org.scalacheck.rng.Seed$"
+      "condition" : {
+        "typeReached" : "org.scalacheck.rng.Seed$"
       },
-      "type": "java.lang.Object[][]"
+      "type" : "java.lang.Object[][]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$2"
+      "condition" : {
+        "typeReached" : "org.scalacheck.ScalaCheckRunner$$anon$2"
       },
-      "type": "java.lang.String[]"
+      "type" : "java.lang.String[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$3"
+      "condition" : {
+        "typeReached" : "org.scalacheck.ScalaCheckRunner$$anon$3"
       },
-      "type": "java.lang.String[]"
+      "type" : "java.lang.String[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalacheck.Platform$"
+      "condition" : {
+        "typeReached" : "org.scalacheck.ScalaCheckRunner"
       },
-      "type": "org_scalacheck.scalacheck_2_13.PlatformClassProperties"
+      "type" : "sbt.testing.Task[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalacheck.Platform$"
+      "condition" : {
+        "typeReached" : "org.scalacheck.ScalaCheckRunner$$anon$2"
       },
-      "type": "org_scalacheck.scalacheck_2_13.PlatformModuleProperties$"
+      "type" : "sbt.testing.Task[]"
     },
     {
-      "condition": {
-        "typeReached": "org.scalacheck.ScalaCheckRunner"
+      "condition" : {
+        "typeReached" : "org.scalacheck.ScalaCheckRunner$$anon$3"
       },
-      "type": "sbt.testing.Task[]"
-    },
-    {
-      "condition": {
-        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$2"
-      },
-      "type": "sbt.testing.Task[]"
-    },
-    {
-      "condition": {
-        "typeReached": "org.scalacheck.ScalaCheckRunner$$anon$3"
-      },
-      "type": "sbt.testing.Task[]"
+      "type" : "sbt.testing.Task[]"
     }
   ]
 }

--- a/metadata/org.scalacheck/scalacheck_2.13/index.json
+++ b/metadata/org.scalacheck/scalacheck_2.13/index.json
@@ -1,6 +1,19 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "1.19.0",
+    "language" : {
+      "name" : "scala",
+      "version" : "2"
+    },
+    "tested-versions" : [
+      "1.19.0"
+    ],
+    "allowed-packages" : [
+      "org.scalacheck"
+    ]
+  },
+  {
     "metadata-version" : "1.18.1",
     "source-code-url" : "https://repo.maven.apache.org/maven2/org/scalacheck/scalacheck_2.13/$version$/scalacheck_2.13-$version$-sources.jar",
     "repository-url" : "https://github.com/typelevel/scalacheck",

--- a/metadata/org.scalacheck/scalacheck_2.13/index.json
+++ b/metadata/org.scalacheck/scalacheck_2.13/index.json
@@ -2,6 +2,11 @@
   {
     "latest" : true,
     "metadata-version" : "1.19.0",
+    "source-code-url" : "https://repo.maven.apache.org/maven2/org/scalacheck/scalacheck_2.13/$version$/scalacheck_2.13-$version$-sources.jar",
+    "repository-url" : "https://github.com/typelevel/scalacheck",
+    "test-code-url" : "https://github.com/typelevel/scalacheck/tree/v$version$/core/shared/src/test",
+    "documentation-url" : "https://repo.maven.apache.org/maven2/org/scalacheck/scalacheck_2.13/$version$/scalacheck_2.13-$version$-javadoc.jar",
+    "description" : "ScalaCheck is a Scala property-based testing library for generating test data and checking properties over many randomized cases. It can be used standalone or with Scala test frameworks to exercise Scala and Java code and shrink failing examples.",
     "language" : {
       "name" : "scala",
       "version" : "2"

--- a/stats/org.scalacheck/scalacheck_2.13/1.19.0/execution-metrics.json
+++ b/stats/org.scalacheck/scalacheck_2.13/1.19.0/execution-metrics.json
@@ -1,0 +1,101 @@
+{
+  "fix_java_run_fail:2026-05-07": {
+    "timestamp": "2026-05-07T03:40:52.487063Z",
+    "library": "org.scalacheck:scalacheck_2.13:1.19.0",
+    "starting_commit": "6474a9252cdd4069c5f37efab2284ce15e62aaeb",
+    "ending_commit": "f796fc25963f9dd566ef41f0e29baa33418d274f",
+    "previous_library": "org.scalacheck:scalacheck_2.13:1.18.1",
+    "strategy_name": "java_run_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "1.19.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 6,
+            "totalCalls": 6
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 6,
+        "totalCalls": 6
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 3757,
+          "missed": 57916,
+          "ratio": 0.060918,
+          "total": 61673
+        },
+        "line": {
+          "covered": 414,
+          "missed": 2526,
+          "ratio": 0.140816,
+          "total": 2940
+        },
+        "method": {
+          "covered": 316,
+          "missed": 3806,
+          "ratio": 0.076662,
+          "total": 4122
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "1.18.1",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 6,
+            "totalCalls": 6
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 6,
+        "totalCalls": 6
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 3702,
+          "missed": 52085,
+          "ratio": 0.06636,
+          "total": 55787
+        },
+        "line": {
+          "covered": 410,
+          "missed": 2501,
+          "ratio": 0.140845,
+          "total": 2911
+        },
+        "method": {
+          "covered": 305,
+          "missed": 3493,
+          "ratio": 0.080305,
+          "total": 3798
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 47943,
+      "cached_input_tokens_used": 235520,
+      "output_tokens_used": 2770,
+      "iterations": 1,
+      "cost_usd": 0.2009,
+      "tested_library_loc": 414,
+      "metadata_entries": 6,
+      "test_only_metadata_entries": 6,
+      "previous_library_metadata_entries": 5,
+      "previous_library_test_only_metadata_entries": 4,
+      "code_coverage_percent": 14.08,
+      "previous_library_coverage_percent": 14.08
+    },
+    "artifacts": {
+      "test_file": "tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/scala/org_scalacheck/scalacheck_2_13/PlatformTest.scala",
+      "metadata_file": "metadata/org.scalacheck/scalacheck_2.13/1.19.0/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/org.scalacheck/scalacheck_2.13/1.19.0/stats.json
+++ b/stats/org.scalacheck/scalacheck_2.13/1.19.0/stats.json
@@ -1,0 +1,37 @@
+{
+  "versions" : [ {
+    "version" : "1.19.0",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "reflection" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 6,
+          "totalCalls" : 6
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 6,
+      "totalCalls" : 6
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 3757,
+        "missed" : 57916,
+        "ratio" : 0.060918,
+        "total" : 61673
+      },
+      "line" : {
+        "covered" : 414,
+        "missed" : 2526,
+        "ratio" : 0.140816,
+        "total" : 2940
+      },
+      "method" : {
+        "covered" : 316,
+        "missed" : 3806,
+        "ratio" : 0.076662,
+        "total" : 4122
+      }
+    }
+  } ]
+}

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/.gitignore
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/build.gradle
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/build.gradle
@@ -1,0 +1,36 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+    id "scala"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "org.scalacheck:scalacheck_2.13:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+    testImplementation 'org.scala-lang:scala-library:2.13.16'
+    testImplementation 'org.scala-lang:scala-compiler:2.13.16'
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)
+    }
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/gradle.properties
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = org.scalacheck:scalacheck_2.13:1.19.0
+library.version = 1.19.0
+metadata.dir = org.scalacheck/scalacheck_2.13/1.19.0/

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/settings.gradle
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'org.scalacheck.scalacheck_2.13_tests'

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -22,6 +22,18 @@
           "name" : "MODULE$"
         }
       ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalacheck.Platform$"
+      },
+      "type" : "org_scalacheck.scalacheck_2_13.PlatformClassProperties"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalacheck.Platform$"
+      },
+      "type" : "org_scalacheck.scalacheck_2_13.PlatformModuleProperties$"
     }
   ]
 }

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -1,0 +1,27 @@
+{
+  "reflection" : [
+    {
+      "condition" : {
+        "typeReached" : "org.scalacheck.Platform$"
+      },
+      "type" : "org_scalacheck.scalacheck_2_13.PlatformClassProperties",
+      "methods" : [
+        {
+          "name" : "<init>",
+          "parameterTypes" : [ ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.scalacheck.Platform$"
+      },
+      "type" : "org_scalacheck.scalacheck_2_13.PlatformModuleProperties$",
+      "fields" : [
+        {
+          "name" : "MODULE$"
+        }
+      ]
+    }
+  ]
+}

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/scala/org_scalacheck/scalacheck_2_13/PlatformTest.scala
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/scala/org_scalacheck/scalacheck_2_13/PlatformTest.scala
@@ -1,0 +1,100 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_scalacheck.scalacheck_2_13
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.scalacheck.Prop
+import org.scalacheck.Properties
+import org.scalacheck.ScalaCheckFramework
+import sbt.testing.Event
+import sbt.testing.EventHandler
+import sbt.testing.Fingerprint
+import sbt.testing.Logger
+import sbt.testing.Selector
+import sbt.testing.Status
+import sbt.testing.SubclassFingerprint
+import sbt.testing.Task
+import sbt.testing.TaskDef
+
+import scala.collection.mutable.ListBuffer
+
+class PlatformTest {
+  @Test
+  def frameworkLoadsPropertiesModule(): Unit = {
+    val statuses: Seq[Status] = runDiscoveredProperties(
+      "org_scalacheck.scalacheck_2_13.PlatformModuleProperties",
+      isModule = true
+    )
+
+    assertEquals(Seq(Status.Success), statuses)
+  }
+
+  @Test
+  def frameworkInstantiatesPropertiesClass(): Unit = {
+    val statuses: Seq[Status] = runDiscoveredProperties(
+      "org_scalacheck.scalacheck_2_13.PlatformClassProperties",
+      isModule = false
+    )
+
+    assertEquals(Seq(Status.Success), statuses)
+  }
+
+  private def runDiscoveredProperties(fullyQualifiedName: String, isModule: Boolean): Seq[Status] = {
+    val framework: ScalaCheckFramework = new ScalaCheckFramework
+    val fingerprint: Fingerprint = propertiesFingerprint(framework, isModule)
+    val loader: ClassLoader = getClass.getClassLoader
+    val runner: sbt.testing.Runner = framework.runner(
+      Array("-minSuccessfulTests", "1"),
+      Array.empty[String],
+      loader
+    )
+    val taskDef: TaskDef = new TaskDef(fullyQualifiedName, fingerprint, true, Array.empty[Selector])
+    val rootTasks: Array[Task] = runner.tasks(Array(taskDef))
+    val statuses: ListBuffer[Status] = ListBuffer.empty[Status]
+    val handler: EventHandler = (event: Event) => statuses += event.status()
+
+    assertEquals(1, rootTasks.length)
+    val propertyTasks: Array[Task] = rootTasks.flatMap(_.execute(handler, Array(NoOpLogger)))
+    assertEquals(1, propertyTasks.length)
+    val remainingTasks: Array[Task] = propertyTasks.flatMap(_.execute(handler, Array(NoOpLogger)))
+
+    assertTrue(remainingTasks.isEmpty)
+    assertEquals("Passed: Total 1, Failed 0, Errors 0, Passed 1", runner.done())
+    statuses.toSeq
+  }
+
+  private def propertiesFingerprint(framework: ScalaCheckFramework, isModule: Boolean): Fingerprint =
+    framework.fingerprints().collectFirst {
+      case fingerprint: SubclassFingerprint
+          if fingerprint.isModule() == isModule && fingerprint.superclassName() == "org.scalacheck.Properties" =>
+        fingerprint
+    }.getOrElse(throw new AssertionError(s"No ScalaCheck Properties fingerprint found for module=$isModule"))
+}
+
+object NoOpLogger extends Logger {
+  override def ansiCodesSupported(): Boolean = false
+
+  override def error(msg: String): Unit = ()
+
+  override def warn(msg: String): Unit = ()
+
+  override def info(msg: String): Unit = ()
+
+  override def debug(msg: String): Unit = ()
+
+  override def trace(t: Throwable): Unit = ()
+}
+
+object PlatformModuleProperties extends Properties("platformModuleProperties") {
+  property("passes") = Prop.proved
+}
+
+class PlatformClassProperties extends Properties("platformClassProperties") {
+  property("passes") = Prop.proved
+}

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/scala/org_scalacheck/scalacheck_2_13/PlatformTest.scala
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/src/test/scala/org_scalacheck/scalacheck_2_13/PlatformTest.scala
@@ -16,9 +16,9 @@ import sbt.testing.Event
 import sbt.testing.EventHandler
 import sbt.testing.Fingerprint
 import sbt.testing.Logger
-import sbt.testing.Selector
 import sbt.testing.Status
 import sbt.testing.SubclassFingerprint
+import sbt.testing.SuiteSelector
 import sbt.testing.Task
 import sbt.testing.TaskDef
 
@@ -54,7 +54,7 @@ class PlatformTest {
       Array.empty[String],
       loader
     )
-    val taskDef: TaskDef = new TaskDef(fullyQualifiedName, fingerprint, true, Array.empty[Selector])
+    val taskDef: TaskDef = new TaskDef(fullyQualifiedName, fingerprint, true, Array(new SuiteSelector))
     val rootTasks: Array[Task] = runner.tasks(Array(taskDef))
     val statuses: ListBuffer[Status] = ListBuffer.empty[Status]
     val handler: EventHandler = (event: Event) => statuses += event.status()

--- a/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/user-code-filter.json
+++ b/tests/src/org.scalacheck/scalacheck_2.13/1.19.0/user-code-filter.json
@@ -1,0 +1,13 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "org.scalacheck.**"
+    },
+    {
+      "includeClasses" : "org_scalacheck.scalacheck_2_13.**"
+    }
+  ]
+}

